cli/utils/scan_utils.py

```diff
@@ -1,7 +1,14 @@
+from datetime import datetime
+
 import click
 
 
 def is_scan_failed(context: click.Context):
     did_fail = context.obj.get("did_fail")
     issue_detected = context.obj.get("issue_detected")
     return did_fail or issue_detected
+
+
+def print_click_secho(message: str):
```

Comment:
not needed, use logger

Reply:
It's should be printed always not by logger

Reply:
so use logger.info

Reply:
No we show also progress bar it can't be by logger.info

Reply:
Fix